centralised-logging.txt

```diff
@@ -79,33 +80,20 @@ Check the journal for fluentd and ensure it has no errors at start
 
  journalctl -u fluentd -l -f
 
-Finally we will configure kibana to point to our elasticsearch instace. Create the file /etc/httpd/conf.d/kibana3.conf and place the following content inside
+Finally we will configure kibana to point to our elasticsearch instace. Create the file /etc/httpd/conf.d/kibana.conf and place the following content inside
```

Comment:
same here: a note on this is going to configure reverse proxy and opening the firewall to use kibana directly is not necessary.